authors/apps/articles/tests/test_base.py

```diff
@@ -0,0 +1,53 @@
+from authors.apps.authentication.tests.test_base import TestBase
+from django.urls import reverse
+from authors.apps.articles.models import Article
+
+
+class ArticleBaseTest(TestBase):
+    def setUp(self):
+        self.article_url = reverse("articles:articles")
+        super().setUp()
+
+        self.article_data = {
+            "article": {
+                "title": "Most people are good",
+                "description": "they are all good",
+                "body": "It is believed that most poeple love what they sghf gvsdf csbkfdgc cbghfgx df",
+                "image": "https://unsplash.com/photos/BW9ki_tmouE"
+            }
+        }
+
+        self.article_data1 = {
+            "article": {
+                "title": "Most people are good",
+                "description": "they are all gssood",
+                "body": "It is believed that modghgst poeple love what they sghf gvsdf csbkfdgc cbghfgx df",
+                "image": "https://unsplash.com/photos/BW9ki_tmouE"
+            }
+        }
+
+        self.new_db_article = Article(
+            title="Most people are good",
+            description="they are all good",
+            body="It is believed that most poeple love what they sghf gvsdf csbkfdgc cbghfgx df",
+            image="https://unsplash.com/photos/BW9ki_tmouE"
+
+        )
+
+        self.new_db_new_article = Article(
+            title="Most people are good",
+            description="they are all good",
+            body="It is believed that most poeple and yes love what they sghf gvsdf csbkfdgc cbghfgx df",
```

Comment:
Could you please format this file to standards to get rid of the several too long lines in it.

Reply:
I've done that